catroid/res/values-ar/strings.xml

```diff
@@ -391,11 +391,11 @@
   <string name="stage_dialog_maximize">زيادة الامتداد\n</string>
   <string name="stage_dialog_screenshot">لقطة للشاشة</string>
   <!-- Broadcast spinner -->
-  <string name="new_broadcast_message">جديد&#8230;</string>
+  <string name="new_broadcast_message">جديد…</string>
```

Comment:
> ...you should've add the character on the left hand side.

Nah :wink:!
The numeric character reference `&#8230;` is also on the right side (open this line in Android Studio if your browser somehow splits it) and changing the position of the ellipsis is beyond the purpose of this PR.

Your doubt is still valid. I just tested both strings and the `…` is automatically moved to the left side by Android. So I don't know why the Crowdin translator added the three dots on the right side... :confused:

Can you take a look directly at the Crowdin translation? Maybe there is a problem during the integration process.

> ...The best way to go about changing this would be to change the character in the default strings.xml and then add it in Crowdin's system.

Sure! Someone will add them to the Crowdin system for me, right?